python/cuml/cluster/kmeans.pyx

```diff
@@ -534,6 +570,66 @@ class KMeans(Base):
         del(clust_mat)
         return preds_gdf
 
+    def score(self, X):
+        """
+        Calcualate the inertia value.
+
+        Parameters
+        ----------
+        X : array-like (device or host) shape = (n_samples, n_features)
+            Dense matrix (floats or doubles) of shape (n_samples, n_features).
+            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
+            ndarray, cuda array interface compliant array like CuPy
+
```

Comment:
I would include the return value here as well, even if it’s spelled out in the description.